backoffice/templates/backoffice/users/add_card.html

```diff
@@ -0,0 +1,19 @@
+{% extends "backoffice_base.html" %}
+
+{% load crispy_forms_tags %}
+
+{% block heading %}Add card for {{user_details.username|title}} {% endblock %}
+{% block buttons %}
+
+{% endblock %}
+
+{% block main %}
+<div class="w-96 p-5">
+    <form method="POST" class="px-5 py-3" action="{% url "backoffice_add_card" user_details.id %}">
+        {% csrf_token %}
+        {{ form|crispy }}
+        <input type="submit" value="Add Card" class="w-72 flex justify-center py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-teal-600 hover:bg-teal-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"/>
```

Comment:
Testing this form has made me realise that the card serial is missing a unique constraint in the database 🫢 

Probably something to fix in another PR.